### PR TITLE
feat: rename commands

### LINKED ISSUE #419: Rename `/assign` to `/start`, `/unassign` to `/stop`
The intent for this command is to represent when the bounty hunter is actively working on the task, vs trying to just "reserve" the task.

The bounty hunters should only be working on a single task at a time. They should not be able to be "reserving" tasks and then neglecting to do them. 

Example: https://github.com/ubiquity/ubiquity-dollar/issues/616#issuecomment-1613354667

Renaming the command can provide a hint to the bounty hunter that they must actively work on the task when they use the command.

### PR DESCRIPTION
**Pull request by me505**
Resolves #419

### CODE REVIEW

**Comment by me505** [comment]
this might work?
i was not sure with this one tbh

**Comment by me505** [comment]
@rndquu might this work?

**Comment by rndquu** [comment]
> @rndquu might this work?

There are many places in the code where refactor should be performed:
- https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/wildcard/unassign.ts#L75
- https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/comment/handlers/help.ts#L23
- https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/comment/handlers/assign.ts#L19
- https://github.com/ubiquity/ubiquibot/blob/development/src/configs/strings.ts#L4
- etc...

File: src/handlers/comment/commands.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export enum IssueCommentCommands {
   HELP = "/help", // list available commands
-  ASSIGN = "/assign", // assign the hunter to the issue automatically
-  UNASSIGN = "/unassign", // unassign to default
+  ASSIGN = "/start", // assign the hunter to the issue automatically
+  UNASSIGN = "/stop", // unassign to default
```
Comment (0xcodercrane) [inline src/handlers/comment/commands.ts:4]: still seeing `ASSIGN` and `UNASSIGN`. They also need to be replaced

**Review by 0xcodercrane — CHANGES_REQUESTED** [review]

File: src/handlers/comment/commands.ts (hunk 1 above)
Comment (me505) [reply src/handlers/comment/commands.ts:4]: oh sorry about that!

Comment (0xcodercrane) [reply src/handlers/comment/commands.ts:4]: did you push something @me505 ? I don't see any commits after the conversation.

**Review by 0xcodercrane — APPROVED** [review]

**Comment by 0xcodercrane** [comment]
@me505 would you resolve the conflicts?

**Comment by me505** [comment]
ok

**Comment by 0xcodercrane** [comment]
Still failing in build and also seeing conflicts @me505

**Review by 0xcodercrane — APPROVED** [review]

File: src/handlers/wildcard/unassign.ts [hunk 2]
```diff
@@ -72,7 +72,8 @@ const checkBountyToUnassign = async (issue: Issue): Promise<boolean> => {
         );
       } else
         await addCommentToIssue(
-          `${askUpdate} @${assignees[0]}? If you would like to release the bounty back to the DevPool, please comment \`/unassign\` \nLast activity time: ${lastActivity}`,
+          `${askUpdate} @${assignees[0]}? If you would like to release the bounty back to the DevPool, please comment \`
```
Comment (pavlovcik) [inline src/handlers/wildcard/unassign.ts:75]: I think there is a mistake here @me505 @0xcodercrane

Comment (0xcodercrane) [reply src/handlers/wildcard/unassign.ts:75]: good catch!

Comment (0xcodercrane) [reply src/handlers/wildcard/unassign.ts:75]: fixed. ➡️ https://github.com/ubiquity/ubiquibot/commit/b742637e3f95b91e0df0b36f0735d05f5b746dac

**Pull request merged**